lib/tasks/widgets/tasks_screen.dart

```diff
@@ -107,6 +120,19 @@ class _TasksScreenState extends State<TasksScreen> {
                           )
                         ],
                       ),
+                      //bla
+                      onTap: () => Navigator.of(context)
+                          .push(MaterialPageRoute<Task>(
+                        builder: (context) => NewTaskDialog(task: t),
+                      ))
+                          .then((val) {
+                        if (val == null) return;
+                        Provider.of<UserService>(context)
```

Comment:
This is not a very elegant way of editing the saved task. Consider making the relevant fields of the `Task` class mutable.